tutorials/commands/flux-jobs.rst

```diff
@@ -0,0 +1,344 @@
+.. _flux-jobs:
+
+================================================================
+How to List Flux jobs, Filter Them, and Output Extra Information
+================================================================
+
+Inevitably you will want to list your current jobs.  Many users will know that
+to list jobs you use the ``flux jobs`` command.  However, there are many
+advanced job listing and filtering options available.  This tutorial will go
+through some of the most popular ones so you know how to list the jobs you are
+looking for more easily and to get the information you are looking for.
+
+--------------------
+Job Submission Setup
+--------------------
+
+Before beginning the tutorial, lets run some jobs to set things up.  I have a small Flux instance of
```

Comment:
I think "lets" should be "let's" here.

Reply:
Er, that doesn't sound right.  Here lets is like "allows", as opposed to "let us" which is contracted to "let's".

If in doubt switch to "allows" :smile:

Reply:
Hmmm.  I'm not reading it that way ... "let us run some jobs" sounds correct, like "how about we run some jobs"?  vs. "you have permission to run some jobs"?

Reply:
Oh sorry!  My fault!